Sources/Apodini/Modifier/TypedRelationshipDestinationModifier.swift

```diff
@@ -0,0 +1,39 @@
+//
+// Created by Andreas Bauer on 20.01.21.
+//
+
+public struct TypedRelationshipDestinationModifier<H: Handler, To>: HandlerModifier {
```

Comment:
Not sure about that. As this would explicitly state that some type of Response<String> would be okay to pass. Even if this would be currently allowed, it is still semantically somewhat weird in my opinion.

Reply:
nit: Under what circumstances would we mark a relationship to an object that is not returned by a handler?
I think this is a fair constraint

Reply:
As discussed in the work meeting: We will leave it as is.

Reply:
As discussed in the meeting we will keep the `To` generic type unconstrained.